util.py

```diff
@@ -238,9 +238,13 @@ def get_default(field):
     if field == 'identity':
         return uuid.uuid4()
     elif field == 'timestamp_ms':
-        return -2e48 - 1
+        #this int64 field will be a float64 sometimes too (such as in the VBO
+        #that is rendered in the frontend) and we want to keep the sub-second
+        #precision. There are 52 bits in the mantissa, so under that by some margin
+        return -2**48 - 1
     elif field == 'endtime_ms':
-        return 2e48 - 1
+        #See timestamp_ms notes about float64
+        return 2**48
```

Comment:
The idea was that these would be the same value such that when defaulted `timestamp * -1 == endtime_ms`.  This should be 2e48-1.